makani/models/model_package.py

```diff
@@ -145,6 +145,14 @@ def out_channels(self):
     def timestep(self):
         return self.params.dt * self.params.dhours
 
+    def replace_state(self, replace_state=True):
```

Comment:
Why do we pass a replace_state variable and do not use it later? We should either use it in the call or remove it. If we keep it, please rename that routine to update_state.